src/session_impl.cpp

```diff
@@ -543,6 +544,8 @@ void apply_deprecated_dht_settings(settings_pack& sett, bdecode_node const& s)
 #ifdef TORRENT_USE_OPENSSL
 		error_code ec;
 		m_ssl_ctx.set_verify_mode(boost::asio::ssl::context::verify_none, ec);
+#endif
+#ifdef TORRENT_SSL_PEERS
 #if OPENSSL_VERSION_NUMBER >= 0x90812f
```

Comment:
this test can be removed now, right?

Reply:
and any similar tests, except for the one you added for the `#error`

Reply:
yes, I will remove them